deliver/lib/deliver/runner.rb

```diff
@@ -72,10 +72,10 @@ def upload_metadata
     def prepare_app_icons(options = {})
       return unless options[:metadata_path]
 
-      default_app_icon_path = File.join(options[:metadata_path], "app_icon.png")
+      default_app_icon_path = Dir["#{options[:metadata_path]}/app_icon.{png,jpg}"].first
```

Comment:
@lacostej yeah i think almost nobody will download over existing. - so first should be fine.
but i did a download a few times, on apps where customers changed it via itc and i had to sync git repo (don't ask)